src/serializer/ResidualHeapSerializer.js

```diff
@@ -1692,13 +1692,26 @@ export class ResidualHeapSerializer {
     let remainingProperties = new Map(val.properties);
     const dummyProperties = new Set();
     let props = [];
+    let isFunction = f => f instanceof ECMAScriptSourceFunctionValue;
```

Comment:
It is neither clear to me that it is correct to just check for instances of ECMAScriptSourceFunctionValue nor that it is a good idea to name this check "isFunction".